spec/order_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'order'
+
+describe Order do
+
+  let(:menu_klass) {double :menu_klass}
+  let(:menu) {double :menu}
+
+  subject(:order) {described_class.new(:menu)}
+
+  before do
+    allow(menu).to receive(:new)
+    allow(menu).to receive(:duplicate_menu).and_return :item_1 => 2, :item_2 => 2
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)